src/widgets/Notebook.hpp

```diff
@@ -19,6 +21,15 @@ class SplitContainer;
 
 enum NotebookTabLocation { Top = 0, Left = 1, Right = 2, Bottom = 3 };
 
+// Controls the visibility of tabs in this notebook
+enum NotebookTabVisibility : int {
+    // Show all tabs
+    Default = 0,
```

Comment:
Should probably be renamed to `AllTabs` - it just happens to be the default setting